lib/dicts.bzl

```diff
@@ -38,6 +38,33 @@ def _add(*dictionaries, **kwargs):
     result.update(kwargs)
     return result
 
+def _omit(dictionary, keys):
+    """Returns a new `dict` that has all the entries of `dictionary` with keys not in `keys`.
+
+    Args:
+      dictionary: A `dict`.
+      keys: A `list` of `str`.
```

Comment:
To expand the point a bit: dict keys don't have to be strings. And there's no reason to require `keys` parameterto be a list (it could easily be a tuple; one can image case where you might naturally use a tuple here - e.g. `native.existing_rule("rule_name")["attr_name"]`).